backend/routes/userInfo.js

```diff
@@ -4,15 +4,16 @@ const {google} = require('googleapis')
 const { GOOGLE_CLIENT_ID, GOOGLE_CLIENT_SECRET} = require('../auth-keys')
 const axios = require('axios')
 
-const Parse = require('parse/node')
+const Parse = require('parse/node');
+const { time } = require('console');
 Parse.initialize("WrhhT0n3PD3RkdESL6pAsvqN86YDNS9eP0v1VdZg", "WliFyOgGrffxxYv0IfvChkvx8a1ByDYKY7tadIDW")
```

Comment:
whart is this Parse initialize? Is it a secret?

Reply:
yeah it was some auth keys for my parse account.. just removed them and replaced them with variables